fiftyone/plugins/context.py

```diff
@@ -88,18 +88,19 @@ def can_register(self, instance):
 
         return self.plugin_definition.can_register_operator(instance.name)
 
-    def register(self, cls):
+    def register(self, cls, builtin=False):
         """Registers the given operator on the plugin.
 
         .. note::
 
             Any errors are logged rather than being raised.
 
         Args:
-            cls: an :class:`fiftyone.operators.operator.Operator` class
+            cls: an :class:`fiftyone.operators.operator.Operator` or :class:`fiftyone.operators.panel.Panel` class
+            builtin (False): For internal use. Whether the cls is a builtin operator or panel
         """
         try:
-            instance = cls()
+            instance = cls(_builtin=builtin)
```

Comment:
this is unnecessary to expose since you have BUILTIN_PLUGINS_DIR defined and iterate through that.